src/Bicep.Core.Samples/Files/Parameters_LF/main.bicepparam

```diff
@@ -0,0 +1,58 @@
+﻿/*
+This is a
+multiline comment!
+*/
+
+// This is a single line comment
+
+// using keyword for specifying a Bicep file
+using './main.bicep/'
+
+// parameter assignment to literals
+param myInt = 42
+param myStr = "hello world!"
+param myBool = true
+
+// parameter assignment to objects
+param myObj = {
+	name: 'vm1'
+	location: 'westus'
+}
+param myComplexObj = {
+	enabled: true
+	name: 'complex object!'
+	priority: 3
+	data: {
+		a: 'b'
+		c: [
+			'd'
+			'e'
+		]
+	}
+}
+
+// parameter assignment to arrays
+param myIntArr = [
+	1
+	2
+	3
+	4
+	5
+]
+param myStrArr = [
+	'ant'
+	'bear'
+	'cat'
+	'dog'
+]
+param myComplexArr = [
+	'eagle'
+	21
+	false
+	{
+		f: [
+			'g'
+			'h'
+		]
+	}
+]
```

Comment:
Since the we allow non-literal values at syntax level, we can add some test cases to ensure they can also be parsed, for example:

```
param myFunction = union({}, {})

param myComplexArrWithFunction = [
  {
    foo: resourceGroup()
  }
  true
  [
    42
  ]
]
```

They can also be useful for validating semantic diagnostics later, since we are going to implement semantic checking to block non-literal values.